mentorship/gsod/2023/README.md

```diff
@@ -6,11 +6,11 @@
 
 ## About OpenPrivacyTech
 
-OpenPrivacyTech is a 501(c)(3) nonprofit open-source organization dedicated to democratizing privacy technology and privacy-enhancing technologies (PETs). Our mission is to make privacy-enhancing technologies (PETs) accessible to everyone through education, research, and advocacy. Our goal is to advance the field of privacy tech by conducting research, identifying emerging trends and technologies, developing standards and infrastructure for the adoption of privacy-enhancing technologies (PETs). We work with researchers, data scientists, developers, policymakers, industry leaders, and organizations to advocate for strong data privacy and protection laws while promoting the adoption of privacy-enhancing technologies (PETs). 
+OpenPrivacyTech is a 501(c)(3) nonprofit open-source organization dedicated to democratizing privacy technology and privacy-enhancing technologies (PETs). Our mission is to make privacy-enhancing technologies (PETs) accessible to everyone through education, research, and advocacy. Our goal is to advance the field of privacy technology by conducting research, identifying emerging trends and technologies, and developing standards and infrastructure for the adoption of privacy-enhancing technologies (PETs). We work with researchers, data scientists, developers, policymakers, industry leaders, and organizations to advocate for strong data privacy and protection laws while promoting the adoption of privacy-enhancing technologies (PETs).
```

Comment:
I think we should keep the term Privacy Tech

Reply:
@souravcipher Verify and let me know if I need to change anything else.